close-bugs-EOL.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+## To generate the BUGSLIST the following command can be used ## `bugzilla query -p GlusterFS -t NEW,OPEN,ON_QA,VERIFIED,ASSIGNED,POST,MODIFIED,RELEASE_PENDING -v <VERSION>[,<VERSION>,...]`
+
+declare BUGSLIST VERSION
+
+if [ "x$DRY_RUN" != "x" ]; then
+  DR="echo"
+fi
+
+check_for_command()
+{
+  env bugzilla --version >/dev/null 2>&1
+  if [ $? -ne 0 ]; then
+    echo "`bugzilla` command is missing"
+    echo "Install `python-bugzilla` before running this script again"
+    exit 1
+  fi
+}
+
+close_bugs()
+{
+	COMMENT="This bug is getting closed because GlusteFS-${VERSION} has reached its end-of-life.
+
+Note: This bug is being closed using a script. No verification has been performed to check if it still exists on newer releases of GlusterFS.
+If this bug still exists in newer GlusterFS releases, please reopen this bug against the newer release.
```

Comment:
Not everyone can "reopen" bugs. It is better to ask for reporting a new bug against a maintained version.